DTS-STN/Service-Canada-Design-System
# Typography compliance

# DS-##(Jira Issue)-A###(component ID)-(component name)

**Reminder: Add all changes for this PR to the CHANGELOG.md**

Main Reviewer: @

#### Description & Background Context:
Added padding-bottom of 8px on H1 style. 

#### Where should the reviewer start?

File name:

#### How should this be tested?

go to typography and ensure that the bottom padding is 8px between text and bottom border. 

#### Are there any breaking changes?
